torchaudio/prototype/functional/_ray_tracing.py

```diff
@@ -0,0 +1,50 @@
+import torch
+import torchaudio
+
+
+def ray_tracing(
+    room: torch.Tensor,
+    source: torch.Tensor,
+    mic_array: torch.Tensor,
+    num_rays: int,  # TODO: find good default
+    e_absorption: float = 0,  # TODO: accept tensor like in ISM
+    scattering: float = 0,  # TODO: accept tensor like in ISM
+    mic_radius: float = 0.5,
+    sound_speed: float = 343,
+    energy_thres: float = 1e-7,
+    time_thres: float = 10,  # 10s
+    hist_bin_size: float = 0.004,  # 4ms
```

Comment:
we can also add air_absorption as the input argument.

Reply:
Should the `air_absorption` parameter be part of the more global `simulate_rir_[hybrid]()` function? From what I understand in PRA it is used here https://github.com/LCAV/pyroomacoustics/blob/b3e36a6451e25bc700bc723e6d267645d43ac10f/pyroomacoustics/room.py#LL2331C2-L2331C3, i.e. it does not impact how the energy histogram is computed?